### ISSUE TITLE
image now required & priceRange is now a warning with a localBusiness object

### ISSUE THREAD

**Issue by joepagan** [khalwat/seomatic#173]
Hey,

https://www.seroundtable.com/google-localbusiness-schema-image-pricerange-23031.html

I'm trying to fix this with the following before the hook:

    {% set seomaticIdentity = seomaticIdentity|merge({
        "priceRange": "£-££"
    }) %}

Though it appears not to add this to everywhere where localBusiness is used, have you any suggestions?

![image](https://cloud.githubusercontent.com/assets/2337910/21350815/60ae9d02-c6b1-11e6-813c-2ce598dd1365.png)

The 3 warnings in the image above are because priceRange is not included in the author object, publisher object & copyright holder object (which are all made from a localBusiness object).

Is there a better global way to deal with these?

**Comment by khalwat** [comment]
Yeah, I'll be rolling in a fix (and a new field) for this in 1.1.40 (out soon-ish)

**Comment by joepagan** [comment]
Fantastic news, thanks!

**Comment by khalwat** [comment]
It's checked into `develop` right now if you want to try it out:

https://github.com/nystudio107/seomatic/tree/develop

**Comment by khalwat** [comment]
btw, things that are _warnings_ are not required (but it's still probably best to have them in). For instance, I don't list a telephone number, and Google warns me for this, but the data is still complete and valid.

**Comment by joepagan** [comment]
Thank you sir, I shall do my best to test this out tomorrow and report back

**Comment by khalwat** [comment]
Released as 1.1.40

**Comment by joepagan** [comment]
Looks like this is now in all the expected places thank you!

I am curious on if you have found more information on what's required for this value. The example on schema.org only has one example: `$$$`, But I wonder if you do something like $-$$$ as a range? 
Also do you know if the dollar symbol makes any difference? I presume it would not if location is set in schema somewhere.

**Comment by khalwat** [comment]
I searched for _hours_ for definitive answers to that question. I came up largely empty. Schema.org defines the vocabulary, Google uses it, but it's entirely up to Google what they deem as being "required" and "recommended".

They recently switched to _recommending_ `priceRange` -- but until they make a formal statement of what they want in there, we're left guessing. So I picked the safe route, and went with what schema.org recommends, and what I've seen used in the past, which is a simple $ / $$ / $$$ / $$$$ / $$$$$ indicator.

I'm not sure the actual symbol matters much, though I can understand you'd want it in a local currency. I think to search engines, the symbol is irrelevant, so going with a "universal" (or at least widely known) symbol of money is likely best.
